lib/logger.js

```diff
@@ -141,13 +141,20 @@ function Logger(key, options) {
     };
 
     const useHttps = configs.AGENT_PROTOCOL === 'https';
+    let headers;
+
+    if (typeof window === 'object') {
+        headers = Object.assign(configs.DEFAULT_REQUEST_HEADER, {'source': `browser/${pkg.version}`});
+    } else {
+        headers = Object.assign(configs.DEFAULT_REQUEST_HEADER, {
+            'user-agent': options.UserAgent || configs.DEFAULT_USER_AGENT
+        });
+    }
 
     this._req = {
         auth: { username: key }
         , agent: useHttps ? new Agent.HttpsAgent(configs.AGENT_SETTING) : new Agent(configs.AGENT_SETTING)
-        , headers: Object.assign({}, clone(configs.DEFAULT_REQUEST_HEADER), typeof window === 'undefined' && {
-            'user-agent': options.UserAgent || configs.DEFAULT_USER_AGENT
-        })
+        , headers: headers
```

Comment:
here we are defining the `headers` regardless of the fact whether it's used on `browser` or not but I'm just wondering whether this may cause an issue or not

Reply:
headers must be defined and they used to be always defined. Setting user-agent on headers causes issues.